src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -3624,13 +3642,18 @@ namespace AirflowNetworkBalanceManager {
                     }
 
                     // Surface view factor
-                    if (Numbers(surfNum + 2) > 1) {
-                        ShowContinueError("View factor for surface " + Alphas(surfNum + 1) +
+                    if (!Surface(this_VF_object.LinkageSurfaceData(surfNum).SurfaceNum).HeatTransSurf) {
+                        ShowWarningError("Surface=" + Alphas(surfNum + 1) +
+                            " is not a heat transfer surface. Check input in: " + CurrentModuleObject + " " + this_VF_object.LinkageName);
+                        ShowContinueError("Using value of 0 for view factor");
+                        this_VF_object.LinkageSurfaceData(surfNum).ViewFactor = 0;
```

Comment:
Reject non-heat transfer surfaces for AirflowNetwork:Distribution:DuctViewFactors. This includes air boundary surfaces.